Makefile

```diff
@@ -1,6 +1,7 @@
 # Define the target to run if make is called with no arguments.
 default: check
 
+export LOG_LEVEL?=9
```

Comment:
Why set this here? The default logging level should be the default. Tests should be silent (by default) unless they fail, so failures are not hidden in a lot of noisy success. When a dev wants to focus on debugging specific tests they can turn up the noise as needed.

Otherwise LGTM.

Reply:
this is basically "status quo" where we had LOG_LEVEL=fatal.  I propose we merge as is to get the log library updated and avoid "retest" and update it with a follow on pr.  Tests actually only spit out these log messages IF there is a failure.